skaffold.yaml

```diff
@@ -1129,6 +1162,59 @@ profiles:
         - name: gardener-resource-manager
           repository: localhost:5001/eu_gcr_io_gardener-project_gardener_resource-manager
           tag: "{{.IMAGE_TAG3}}"
+- name: extensions
+  patches:
+  - op: replace
+    path: /deploy/helm/hooks/after/0/host/command
+    value:
+    - bash
+    - -ec
+    - |
+      echo "Wait until seed is ready"
+      for i in `seq 1 60`;
+      do
+        if kubectl --kubeconfig=$GARDENER_LOCAL_KUBECONFIG get seed $SEED_NAME 2> /dev/null; then
+          break
+        fi
+        echo "Wait until seed gets created by gardenlet"
+        sleep 2
+      done
+      kubectl --kubeconfig=$GARDENER_LOCAL_KUBECONFIG wait --for=condition=gardenletready --for=condition=bootstrapped seed $SEED_NAME --timeout=5m
+  - op: replace
+    path: /deploy/helm/releases/0/valuesFiles/0
+    value: example/gardener-local/gardenlet/values-provider-extensions.yaml
+    # for the provider-extensions profile only 2 images are built: gardenlet and resource-manager
+    # so the imageVectorOverwrite must be patched to specify the correct tags
+  - op: replace
+    path: /deploy/helm/releases/0/setValueTemplates
+    value:
+      config:
+        seedConfig:
+          metadata:
+            name: "{{.SEED_NAME}}"
+          spec:
+            dns:
+              provider:
+                secretRef:
+                  name: "{{.INTERNAL_DNS_SECRET}}"
+                type:  "{{.DNS_PROVIDER_TYPE}}"
+            ingress:
+              domain: "{{.INGRESS_DOMAIN}}"
+            networks:
+              pods: "{{.PODS_CIDR}}"
+              nodes: "{{.NODES_CIDR}}"
+              services: "{{.SERVICES_CIDR}}"
+            provider:
+              region: "{{.REGION}}"
+              type: "{{.TYPE}}"
```

Comment:
Where are these values coming from again?

Reply:
They are set in `example/provider-extensions/seed/seed-config.yaml`.
For Gardener Shoots promoted to a Seed you could leave most of the values empty. In this case, the values will be copied from kube-system/shoot-info `ConfigMap`. For a non-Gardener seed cluster you must set all values.